make-debs.sh

```diff
@@ -23,17 +23,17 @@ rm -fr $(dirname $releasedir)
 #
 # remove all files not under git so they are not
 # included in the distribution.
-#
-git clean -dxf
-#
+
+[ -d .git ] && git clean -dxf
```

Comment:
I don't think we care much about this use-case, but I recently learned that if you are using git worktrees the `.git` will be a file rather than a directory. I had to adjust a bunch of my homegrown scripts for this. I just felt like sharing this story, no need to change this for this PR. :-)

Reply:
That's a good catch and I think I'd rather just use `-e` to handle both cases